src/JiraCli/Services/VersionInfoService.cs

```diff
@@ -95,6 +95,40 @@ private VersionComparisonResult ConvertToComparisonResult(int result)
 
         }
 
+        public bool IsPreRelease(string version, Predicate<string> prereleaseLabelChecker)
+        {
+            Argument.IsNotNull(() => version);
+
+            // Can assume semver format.
+            SemVersion semVer;
+            if (SemVersion.TryParse(version, out semVer))
+            {
+                if (!string.IsNullOrWhiteSpace(semVer.Prerelease))
+                {
+                    return prereleaseLabelChecker(semVer.Prerelease);
+                }
+            }
+
+            return false;
+        }
+
+        public bool IsPreRelease(string version)
+        {
+            Argument.IsNotNull(() => version);            
+
+            // Can assume semver format.
+            SemVersion semVer;
+            if (SemVersion.TryParse(version, out semVer))
+            {
+                if (!string.IsNullOrWhiteSpace(semVer.Prerelease))
```

Comment:
I double checked, I commented on the wrong line. See IsPreRelease(stringVersion) instead.